static/usage.html

```diff
@@ -1208,7 +1208,7 @@ <h2><a href="#android-auto">Android Auto</a></h2>
                 allowed notification access. The notification access settings are linked below the
                 permission toggles.</p>
 
-                <p>In order to show up in the Android Auto car interface, apps have to be installed
+                <p>For Android Auto to show up in the car interface, apps have to be installed
```

Comment:
I think the original is more accurate. This section is about apps showing up, not Android Auto.

Reply:
Oh ok, I just wanted to make sentences less wordy tbf. In that case yeah, you can keep the original.